### PR TITLE
MINERVA-2414: fixing CVE issues on galaxy:1.0.11-486

### PR DESCRIPTION
**Pull request by sarmuru2**
Fixing cve issues on https://jira-eng-sjc11.cisco.com/jira/browse/MINERVA-2414?filter=-1
all jars mentioned in the jira, are upgraded. libgnutls30 is updated on build version 1.0.11-491 itself.

Pull Request type
----
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [ ] Other (please describe):

**NOTE**: Please remember to run `./gradlew spotlessApply` to fix any format violations.

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
Issue #

Alternatives considered
----

_Describe alternative implementation you have considered_

### CODE REVIEW

File: es7-persistence/build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ dependencies {
     testImplementation project(':conductor-common').sourceSets.test.output
 
     //Dependencies for signing ES requests with AWS keys
-    compile 'com.amazonaws:aws-java-sdk-s3:1.11.228'
+    compile 'com.amazonaws:aws-java-sdk-s3:1.12.261'
```
Comment (charybr) [inline es7-persistence/build.gradle:49]: com.amazonaws:aws-java-sdk-s3:1.12.261 is old (Jul 15, 2022). How about using 1.12.416 (Feb 28, 2023) ?

File: es7-persistence/dependencies.lock [hunk 2]
```diff
@@ -438,7 +438,7 @@
             ]
         },
         "org.yaml:snakeyaml": {
-            "locked": "1.33",
+            "locked": "2.0",
```
Comment (charybr) [inline es7-persistence/dependencies.lock:441]: snakeyaml 2.0 is fine

File: server/dependencies.lock [hunk 3]
```diff
@@ -1826,7 +1826,7 @@
             ]
         },
         "org.codehaus.jettison:jettison": {
-            "locked": "1.5.3",
+            "locked": "1.5.4",
```
Comment (charybr) [inline server/dependencies.lock:1829]: jettison 1.5.4 is fine

File: es7-persistence/build.gradle (hunk 1 above)
Comment (sarmuru2) [reply es7-persistence/build.gradle:49]: we have 37 usages for 1.12.261 but we have very less usages for 1.1.2.416

Comment (charybr) [reply es7-persistence/build.gradle:49]: Using 1.12.261 is fine for now.

**Review by charybr — APPROVED** [review]

**Pull request merged**